gptme/eval/run.py

```diff
@@ -229,6 +230,23 @@ def execute(
             gen_stderr = result.get("stderr", "")
             log_dir = result.get("log_dir") or agent.log_dir
             workspace_dir = result.get("workspace_dir") or agent.workspace_dir
+
+            # Extract cost from subprocess result
+            cost_dict = result.get("cost")
+            cost = None
+            if cost_dict:
+                from .cost import CostSummary
+
+                cost = CostSummary(
+                    session_id=cost_dict["session_id"],
+                    total_cost=cost_dict["total_cost"],
+                    total_input_tokens=cost_dict["total_input_tokens"],
+                    total_output_tokens=cost_dict["total_output_tokens"],
+                    cache_read_tokens=cost_dict["cache_read_tokens"],
+                    cache_creation_tokens=cost_dict["cache_creation_tokens"],
+                    cache_hit_rate=cost_dict["cache_hit_rate"],
+                    request_count=cost_dict["request_count"],
+                )
```

Comment:
This can't be the right way to do this, the class might need a from_dict which type checks or just kwarg it all.